inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1193,72 +1502,95 @@ This resulted in `r fmt_count(study_data_cleaned, endcause == "Present at end")`
+starting and end points during the study period. 
+
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/// visualise_population_flow \\\
+--------------------------------------------------------------------------------
 
-```{r visualise_population_flow, warning = FALSE, fig.width = 12}
+The below chunk creates a parallel set diagram in two steps: 
+- prepare appropriately formatted counts table by start and end cause 
+- plot these on a diagram
 
-# summarize data
+Note that this does not account for time of arrival/departure - and only 
+shows arbitrary arrival and departure reasons. 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+
+```{r visualise_population_flow, warning = FALSE, fig.width = 12, fig.height = 10}
+
+## summarize data
 flow_table <- study_data_cleaned %>%
   count(startcause, endcause, sex) %>%  # get counts 
-  gather_set_data(x = c("startcause", "endcause")) %>%  # change to appropriate format for plotting
-  mutate(x = fct_relevel(x, c("startcause", "endcause")),  # set the factor levels to have start first
+  gather_set_data(x = c("startcause", "endcause")) %>%     # change format for plotting
+  mutate(x = fct_relevel(x, c("startcause", "endcause")),  # set startcause as first level
          x = fct_recode(x, 
-                        "Start \n cause" = "startcause",  # add a return (\n): cause plots below start
+                        "Start \n cause" = "startcause",   # add line break (\n) after start
                         "End \n cause"   = "endcause")
         )
 
-# plot your dataset 
-  # on the x axis is the start and end causes
-  # id is generated by gather_set_data and ensures each individual is combo is plotted seperate
-  # splitting by y gives the possible start/end combos
-  # value as n gives it as counts (could also be changed to proportion if manipulate)
+## plot your dataset 
+  ## on the x axis is the start and end causes
+  ## gather_set_data generates an ID for each possible combination
+  ## splitting by y gives the possible start/end combos
+  ## value as n gives it as counts (could also be changed to proportion)
 ggplot(flow_table, aes(x, id = id, split = y, value = n)) +
-  # colour lines by sex 
+  ## colour lines by sex 
   geom_parallel_sets(aes(fill = sex), alpha = 0.5, axis.width = 0.2) +
-  # fill in the label boxes grey
+  ## fill in the label boxes grey
   geom_parallel_sets_axes(axis.width = 0.15, fill = "grey80", color = "grey80") +
-  # change text colour and angle (needs to be adjusted)
+  ## change text colour and angle (needs to be adjusted)
   geom_parallel_sets_labels(color = "black", angle = 0) +
-  # adjusted y and x axes (probably needs more vertical space)
+  ## adjusted y and x axes (probably needs more vertical space)
   scale_x_discrete(name = NULL, expand = c(0, 0.2)) +
   scale_y_continuous(breaks = NULL, expand = c(0, 0)) +
-  # remove axis labels
+  ## remove axis labels
```

Comment:
@zkamvar how do we make the text size bigger here?

Reply:
I assume you mean text size of the labels? It would be to add "size" to the labels geom.

```r
  geom_parallel_sets_labels(color = "black", angle = 0, size = 5) +
```

Reply:
ehmmm yeah just all the text - same way as basesize = 18 in theme_set does

Reply:
The other text should follow the theme. The labels themselves are geom so are not part of the theme. Let me check how it looks when I render the template.